### PR TITLE
fix(gcp): use "custom-4-32768-ext" machine type for gcp

### PR DESCRIPTION
**Pull request by machi1990**
Closes # 

### Verification Steps

1. Do x
2. Do y

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation change
- [ ] Other (please specify)

### CODE REVIEW

**Comment by machi1990** [comment]
ping @shawkins can you've a look as well and confirm the machine types for each cloud provider. Thanks

**Comment by machi1990** [comment]
ping @shawkins can you've a look as well and confirm the machine types for each cloud provider. Thanks

**Review by dimakis — APPROVED** [review]
LGTM 
great changes, Thank you @machi1990

File: pkg/cmd/kafka/openshift-cluster/register/registercluster.go [hunk 1]
```diff
@@ -174,9 +181,7 @@ func setOrCreateMachinePoolList(opts *options) error {
 			return err
 		}
 	} else {
-		for _, machinePool := range response.Items().Slice() {
-			opts.existingMachinePoolList = append(opts.existingMachinePoolList, machinePool)
-		}
+		opts.existingMachinePoolList = append(opts.existingMachinePoolList, response.Items().Slice()...)
```
[the remark this answers is not in the document]
Comment (machi1990) [reply pkg/cmd/kafka/openshift-cluster/register/registercluster.go:184]: I was opportunistic. I should done it in a separate commit but hey..so much for a quick fix ;-)

[the remark this answers is not in the document]
Comment (machi1990) [reply pkg/cmd/kafka/openshift-cluster/register/registercluster.go:184]: thanks

**Pull request merged**